src/docs/tutorials/set-up-a-tailscale-subnet-router.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Set up a Tailscale Subnet Router
+---
+
+## What is a Subnet router?
+
+> A subnet router is a device within your tailnet that you use as a gateway that advertises routes for other devices that you want to connect to your tailnet without installing the Tailscale client.
+
+*Source: <a href="https://tailscale.com/kb/1019/subnets" target="_blank">Subnet routers</a> Via Tailscale's Documentation*
+
+In the context of Railway, The "other devices" are the services within a project.
+
+## About this Tutorial
+
+The goal of this tutorial is to enable us to connect to our production Postgres database without ever needing to expose it publicly.
```

Comment:
That feels like a different way to say the same thing?

Reply:
It is, but, https://ptgmedia.pearsoncmg.com/images/9780132101301/samplepages/0132101300.pdf

I don't have a reference specifically, but it's usually best practice to tell them: "In this guide we will accomplish **goal** X" rather than "In this guide we will do X" - very important to tell the user the end state in mind.